Makefile

```diff
@@ -69,11 +69,15 @@ sbom: ## creates sbom for both  npm and hex dependancies
 	mix deps.get && mix sbom.cyclonedx -o elixir_bom.xml
 	cd assets/  && npm install && npm install -g @cyclonedx/bom@2.0.2 && cyclonedx-bom -o ../$(SBOM_FILE_NAME_CY).xml -a ../elixir_bom.xml && cd ..
 	./cyclonedx-cli convert --input-file $(SBOM_FILE_NAME_CY).xml --output-file $(SBOM_FILE_NAME_CY).json
+	cp $(SBOM_FILE_NAME_CY).* assets/static/.well-known/sbom
+	cp $(SBOM_FILE_NAME_CY).json assets/static/.well-known/sbom/sbom.json
 
 sbom_fast: ## creates sbom without dependancy instalment, assumes you have cyclonedx-bom javascript package installed globally
 	mix sbom.cyclonedx -o elixir_bom.xml
 	cd assets/ && cyclonedx-bom -o ../$(SBOM_FILE_NAME_CY).xml -a ../elixir_bom.xml && cd ..
 	./cyclonedx-cli convert --input-file $(SBOM_FILE_NAME_CY).xml --output-file $(SBOM_FILE_NAME_CY).json
+	cp $(SBOM_FILE_NAME_CY).* assets/static/.well-known/sbom
```

Comment:
I'm getting an error when running `make sbom`  couldn't copy to .well-known/sbom directory. This is because I don't have  `.well-known/sbom` directory . I'm guessing there is a step that I must be missing . How can I ensure I have the directory?